includes/definitions/hatteras.yaml

```diff
@@ -0,0 +1,58 @@
+os: hatteras 
+text: 'Hatteras Overdue DSLAM'
+type: dslam
+icon: hatteras
+
+empty_ifdescr: true
+ifname: true
+over:
+    - { graph: device_bits, text: 'Device Traffic' }
+discovery:
+    - sysDescr:
+            - HN4000-I
+            - HN4000e
+bad_if:
+  - notPresent
+  - serConsole
+good_if: 
+  - ras #all interfaces has 'Hatteras' in it...
+discovery_modules:
+    ntp: false
```

Comment:
Hi @Bierchermuesli 

Any reason to force modules being activated and deactivated ? If they don't fail, better to leave default and remove completely the section. If any of them is harming the device, then only disable these. 

Thanx

Reply:
Those modules really don't work with this device, so I hope for performance better to disable. wrong approach?

Reply:
You gain about 2ms in polling time by disabling these and there could be some random user that wants a specific module enabled or someone adds support for another one, and this will prevent that.

We prefer to have them enabled to allow the users full control.  If there is a module that is slow polling when there is no data, then that is a bug.